docs/settingUp.md

```diff
@@ -32,12 +39,16 @@ More information can be found at [this documentation](https://help.github.com/ar
    ```sh
    ./gradlew appengineDownloadSdk
    ```
-
+   
+ **Verification:** Check your Gradle folder (the directory can be found with the command ./gradlew printUserHomeDir).
```

Comment:
Sorry i was still editing the comments (for review) after i noticed the same mistake.